exercises/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+func insertion(s []int) {
+	for i := 1; i < len(s); i++ {
+		for j := i; j > 0; j-- {
```

Comment:
imagine this case: `[1 2 3 5 6 7 8 4]`. everything is sorted but 4, in it is very obvious, that you don't have to iterate until the beginning of the array, and could stop right after `5` is handled. I would recommend fixing this part - let's not make slow sorting algorithms even slower (-;